data/multiplayer/scenarios/ANL_utils/ANL_worker_options.cfg

```diff
@@ -473,7 +473,7 @@
                             image = "terrain/cave/floor.png"
                             label = "<span color='green'>" + _ "Harvest Mushrooms" + "</span>"
                             description = "<span size='small'>" + _ "Earns: 3g" + "</span>"
-                            {ANL_SHOW_IF ("Peasant") "Uu^Uf"}
+                            {ANL_SHOW_IF ("Peasant") "Tb^Tf"}
```

Comment:
I think this needs to be Uu^Tf. @sevu, please would you confirm?

The corresponding map change isn't in this commit, it was already done in 353dce142ebc55ff954e7de66171b80fb51bfffa. Looks like the effect is that peasants can't harvest mushrooms in 1.15.4.

Reply:
Yes. Easy fix is changing it to `Uu^Tf`. Better fix is using `*^Tf` and using 29 lines farther
[terrain]
x,y=$x1,$y1
terrain="^"
layer=overlay
[/terrain]
@Pentarctagon can change it or remove it from the commit

Reply:
@sevu You mean changing `{MODIFY_TERRAIN "Uu" $x1 $y1}` to
```
[terrain]
x,y=$x1,$y1
terrain="^"
layer=overlay
[/terrain]
```
?

Reply:
yes

Reply:
Alright, updated.